oas_docs/output/kibana.yaml

```diff
@@ -40707,6 +40713,149 @@ paths:
       summary: Get value list privileges
       tags:
         - Security Lists API
+  /api/logstash/pipeline/{id}:
+    delete:
+      description: Delete a centrally-managed Logstash pipeline.
+      operationId: delete-logstash-pipeline
+      parameters:
+        - description: An identifier for the pipeline.
+          in: path
+          name: id
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
```

Comment:
don't we need to include other (403, 404, 500, etc..) statuses?

Reply:
More can definitely be added (perhaps in a subsequent PR?). For now I just copied what existed in the docs.